libraries/AP_VideoTX/AP_VideoTX.cpp

```diff
@@ -350,9 +352,9 @@ bool AP_VideoTX::set_defaults()
 void AP_VideoTX::announce_vtx_settings() const
 {
     // Output a friendly message so the user knows the VTX has been detected
-    GCS_SEND_TEXT(MAV_SEVERITY_INFO, "VTX: Freq: %dMHz, Power: %dmw, Band: %d, Chan: %d",
-        _frequency_mhz.get(), has_option(VideoOptions::VTX_PITMODE) ? 0 : _power_mw.get(),
-        _band.get() + 1, _channel.get() + 1);
+    GCS_SEND_TEXT(MAV_SEVERITY_INFO, "VTX: %s%d %dMHz, PWR: %dmW",
```

Comment:
I'm not sure this formatting is an improvement, why remove the labels? I don't think its too long is it?

Reply:
It was too long, it was scrolling right to left making it mostly unreadable when flying

Reply:
I see, you like seeing the message in your goggles? Seems fair - it's pretty cryptic now though. How about something like "VTX: Airwave1/5600Mhz @25mW" would that fit?

Reply:
What does betaflight do? Note that this is not an OSD display but a mavlink message - we send longer messages to be informational so generally anything up to 50 characters is ok. In my view it sounds like what you are really after is an OSD enhancement that displays the compact information.

Reply:
I don't know about betaflight, but INAV has a bunch of OSD elements for VTX. One showing channel/band/power and one for power. 

TBH this PR does in general some optimization
1)Renders the GCS announcement complete and usable on OSD
2)Reduces the length of the message. Less flash needed and less bandwidth needed on the radio link.

Reply:
My point is that we have lots of GCS announcements for other things that are not intended to be optimized for OSD - they are supposed to be mavlink messages that appear in a ground station. Imposing OSD character limits (30 vs 50) in general makes the messages less useful which is why the OSD itself has other ways of representing useful information.

Reply:
Well, be my guest and customise the GCS message as you wish. Thanks!
I have anyhow to build custom firmwares for my vehicles, so it's not a problem doing a local customisation for me.